mypy.ini

```diff
@@ -0,0 +1,12 @@
+[mypy]
+; third-party packages
```

Comment:
we should put as much as possible in pyproject.toml, looks like mypy supports that